MCPClient: refactor archivematicaClient.py

jrwdunham opened the pull request:
Adds docstrings, more Pythonic formatting, and some cosmetic structural
changes to archivematicaClient.py

sevein commented on src/MCPClient/lib/archivematicaClient.py:
```diff
@@ -1,8 +1,34 @@
 #!/usr/bin/env python2
 
+"""Archivematica (Gearman) Client
```
To be accurate, it's a Gearman worker.

sevein commented on src/MCPClient/lib/archivematicaClient.py:
```diff
@@ -21,166 +47,190 @@
-        output = ["Archivematica Client Error!", traceback.format_exc()]
-        exitCode = 1
-        return cPickle.dumps({"exitCode": exitCode, "stdOut": output[0], "stdError": output[1]})
+        return cPickle.dumps({'exitCode': 1,
```
^ this `OSError` was used to control the exceptions raised by subprocess (executeOrRun), should be done inside _process_gearman_job instead?

sevein reviewed (changes requested):
I have a couple of small observations but looking very good! It should target qa/1.x, this is definitely something we want to include!

jrwdunham committed:
MCPClient: refactor archivematicaClient.py

Adds docstrings, more Pythonic formatting, and some cosmetic structural
changes to archivematicaClient.py

jrwdunham commented:
Replaced by https://github.com/artefactual/archivematica/pull/829
